website/app.py

```diff
@@ -119,6 +123,8 @@ def init_app(settings_module='website.settings', set_backends=True, routes=True,
     init_addons(settings, routes)
     build_js_config_files(settings)
 
+    django.setup()
```

Comment:
Is this to make `reverse` work?

Putting this here may lead to this error:

```python
django.core.exceptions.ImproperlyConfigured: Requested setting LOGGING_CONFIG, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings.
```

Would it make sense to set a default for `os.environ['DJANGO_SETTINGS_MODULE']` to `api.base.settings` here?